modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -127,12 +150,17 @@ export default class Tile3DLayer extends CompositeLayer {
     this.props.onTileUnload(tileHeader);
   }
 
-  _updateTileset(tileset3d) {
-    const {timeline, viewport} = this.context;
-    if (!timeline || !viewport || !tileset3d) {
+  _updateTileset(tileset3d, viewports) {
+    if (!viewports) {
```

Comment:
This is a private method. Always explicitly pass `viewports` for readability.

Reply:
Done